tests/sdio/Makefile

```diff
@@ -47,3 +47,11 @@ OBJ_DIRS = $(sort $(dir $(OBJ)))
 $(OBJ): | $(OBJ_DIRS)
 $(OBJ_DIRS):
 	$(MKDIR) -p $@
+
+# Dependency generation
+%.o: %.d
+CFLAGS += -MP -MMD
+DEP := $(OBJ:.o=.d)
+$(DEP):
```

Comment:
Add dummy recipes for dependency files to avoid "no rule to make target" error.

Dependency files are instead created on compiling the object file.